targets/csharp/templates/PlayFabUtil.cs.ejs

```diff
@@ -81,5 +83,78 @@ namespace PlayFab
             }
             return Sb.ToString();
         }
+
+#if NET45 || NETCOREAPP2_0
+        [ThreadStatic]
+        private static StringBuilder _sb;
+        /// <summary>
+        /// A threadsafe way to block and load a text file
+        /// 
+        /// Load a text file, and return the file as text.
+        /// Used for small (usually json) files.
+        /// </summary>
+        public static string ReadAllFileText(string filename)
```

Comment:
This utility was just removed.
It looks like you did a bad merge.
Please look through this diff and make sure you are only adding your own changes.

Reply:
So we are aware of the fact that it was removed but since we need to read a config file in net45 and core22 scenarios, we decided to improve the method by incorporating more `using` statements to dispose of disposables and bring it back so that we can actually read our config file.

Reply:
I feel like if the function exists, it should be available for all platforms possible, not just a very limited subset. That imbalanced limitation was exactly why we removed it.

Reply:
Okay well Net Portable doesn't provide IO and we're not in favor of throwing away the whole concept of reading out config file because we can't do IO on portable or standard 1.1.
I already spoke to you about this. You said allow the user to use these features on 45 and 22 and on portable we'll remove it since we can't do IO. This is exactly what that is.

Reply:
Let's call this one ok. Clearly it's not a choice for you.
I'll talk to Deb about redoing the cross-platform testing.

Reply:
As discussed, we're looping people in and discussing the cross-platform testing.

Reply:
Reverting changes to C# SDK related to local-debugging for GDC.

Reply:
Resolving as changes reverted.

Reply:
Resolving as changes reverted.